README.md

```diff
@@ -4,22 +4,98 @@
-While the CLI utility can be used to perform many common tasks with minimal configuration,
-the Python package can be employed when more extensive customization is necessary.
+# simexpal 
 
-**Features.** simexpal assists users with the following operations
-- Manage instances: **Download instances** from external sources, **collect information** about instances.
-- Manage runs: **Launch** runs and **monitor** their progress, **detect failed runs** and **restart** them, **collect results** of runs.
+_simexpal_ is a python based tool to setup, manage, launch, monitor, and
+evaluate algorithmic experiments. If you can not wait to try it out and dive
+into it, you may find our [quick start
+guide](https://simexpal.readthedocs.io/en/latest/quick_start.html) handy. 
 
-<!-- Write about reproducibility issues -->
+_simexpal_ is mainly developed by the MACSy (Modelling and Analysis of Complex
+Systems) group where graph data is at the core of most projects. Therefore, the
+tool provides some special functionality for graph algorithm experiments.
+However, this does not limit _simexpal_ to be only used for graph algorithm
+experiments, but is a tool that can be used to setup and manage almost all
+algorithmic experiments. 
 
-## Documentation
+Since algorithmic experiments require several tasks in a specific order, the
+goal of this tool is to clearly describe and automatize these ordered, and
+consecutively executed tasks. Firstly, _simexpal_ allows to make quick changes
+to the experiment setup. Secondly, it allows to describe experiments that are
+easy to rerun. When describing the necessary steps, even users with no knowledge
+of _simexpal_ should be able to quickly setup and execute the described
+experiments.
 
-Please refer to the [quick start guide](https://simexpal.readthedocs.io/en/latest/quick_start.html) for further information about
-how to install and use simexpal.
+_simexpal_ consists of both a command line interface (CLI) utility and a python
+package. While the CLI can be used to perform many common tasks with minimal
+configuration, the Python package can be employed when more extensive
+customization is necessary.
 
+## Feature Set Summary
+
+The following is a non-exhaustive list of the functionality provided by
+SimexPal:
+
+- **resolve external resources** such as git repositories
+- describe and setup **build processes**
+- declaratively describe **experiments** the YAML language 
+- define **parameters** and parameter permutations
+- manage **data sets** and specifically graph data
+- **run and monitor** experiments on multiple computing platforms
+- **evaluate** experiment results
+
+## Why should I use SimexPal?
+
+Whether you're a student, an algorithm enthusiast, or a researcher: developing
+experiments that can easily be reproduced will support the validity of your
+findings. Self written scripts often bind experiments to a narrow set of soft-
```

Comment:
I think there should be a hyphen in "Self written": "Self-written".

Reply:
Thanks! :heavy_check_mark: